From 056a6b7c716e76938a065d5c146d5812392bf01f Mon Sep 17 00:00:00 2001
From: Han Xu <b45815@freescale.com>
Date: Tue, 10 Nov 2015 14:19:53 -0600
Subject: [PATCH 0669/5242] MLK-11849: ARM: dts: fix the iomux issue in sai
 dts

commit  f080ed8b4316f610f2367b464574b32f9b2c9bf1 from
https://source.codeaurora.org/external/imx/linux-imx.git

some issue was involved when merging the commit to kernel 4.1

Signed-off-by: Han Xu <b45815@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2-sai.dts |    4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2-sai.dts b/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2-sai.dts
index 9b0d635..f1d36ce 100644
--- a/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2-sai.dts
+++ b/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2-sai.dts
@@ -21,7 +21,9 @@
 };
 
 &iomuxc {
-	pinctrl-0 = <&pinctrl_hog_1>;
+	pinctrl-names = "default", "sleep";
+	pinctrl-0 = <&pinctrl_hog_1 &pinctrl_hog_headphone_det>;
+	pinctrl-1 = <&pinctrl_hog_1 &pinctrl_hog_sd2_vselect>;
 };
 
 &sai1 {
-- 
1.7.9.5

